README.md

```diff
@@ -28,7 +28,7 @@
 
 <h2>Preview of <a href="https://meshery.io">Meshery.io</a> Website<h2>
 
-![Meshery.io website screenshot](images/mesheryio_screenshot.png)
+![Meshery.io website screenshot](assets/images/mesheryio_screenshot.png)
```

Comment:
> > Please update the screenshot of the meshery.io site with the latest one.
> 
> Can you provide me with the image to change with the previously existing image of website screenshot

@Adi-ty, You shouldn't have asked this question, anyways, I've attached the image, update the screenshot with this existing one.

![Screenshot from 2023-04-04 17-45-42](https://user-images.githubusercontent.com/92504666/229788520-421e1942-dffa-46ad-b59e-509377325b0c.png)





> Please update the screenshot of the meshery.io site with the latest one.

and thank you @chinu-anand, I've overlooked the outdated image

Reply:
I have changed the image sorry for inconvenience